src/utils/SearchBoxResize.ts

```diff
@@ -0,0 +1,57 @@
+import { $$ } from './Dom';
+import _ = require('underscore');
```

Comment:
It's preferable to extract the methods needed for underscore.
In this case only forEach is used, which now has a native JS implementation
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

Reply:
Thanks, I'll use that